echo2.cc

```diff
@@ -5,6 +5,7 @@
 
 #include "common/common/assert.h"
 
+namespace Lyft {
```

Comment:
Bah, previous response ended up as an orphaned comment.

I can put it in Envoy if you prefer, but that seems like it's sufficiently generic that there might be other collisions whereas Lyft is unlikely to be grabbed by someone else.